meson.build

```diff
@@ -795,34 +781,31 @@ endif
 libmount = dependency('mount',
                       version : fuzzer_build ? '>= 0' : '>= 2.30')
 
-want_seccomp = get_option('seccomp')
-if want_seccomp != 'false' and not fuzzer_build
+if not fuzzer_build
         libseccomp = dependency('libseccomp',
                                 version : '>= 2.3.1',
-                                required : want_seccomp == 'true')
+                                required : get_option('seccomp'))
```

Comment:
Why just `get_option('seccomp')` instead of `get_option('seccomp').enabled()`? (And same below throughout this file.) You seemed to use an explicit `.enabled()` earlier. Does the simple `get_option('seccomp')` catch both enabled + auto? In that case, why did you use `not get_option('...').disabled()` before, wouldn't that be the same as `get_option('...')` which should cover enabled + auto?

I haven't looked at Meson code to understand it, just the docs, they explain this a bit but not sure whether I grasped all of it...

Reply:
If a feature type object is passed to `request:` and its value is `disabled`, then the dependency object in 'not-found' state.

See, http://mesonbuild.com/Build-options.html#build-options
> disabled do not look for the dependency and always return 'not-found'.